experimental/packages/opentelemetry-sdk-node/package.json

```diff
@@ -60,7 +60,7 @@
     "@opentelemetry/semantic-conventions": "1.25.1"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.3.0 <1.10.0"
+    "@opentelemetry/api": "^1.3.0"
```

Comment:
This needs to stay limited as it needs to re-iterate the limit from the packages it depends on (some package manager don't enforce peer dep ranges of transitive dependencies).